media-plugins/alsaequal/alsaequal-0.7.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+inherit eutils multilib toolchain-funcs multilib-minimal
+
+DESCRIPTION="A real-time adjustable equalizer plugin for ALSA"
+HOMEPAGE="https://github.com/bassdr/alsaequal"
+SRC_URI="https://github.com/bassdr/${PN}/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=media-libs/alsa-lib-1.0.27.2[${MULTILIB_USEDEP}]
+	>=media-plugins/caps-plugins-0.9.15[${MULTILIB_USEDEP}]
+	abi_x86_32? ( !<=app-emulation/emul-linux-x86-soundlibs-20130224-r3
+					!app-emulation/emul-linux-x86-soundlibs[-abi_x86_32(-)] )"
+DEPEND="${RDEPEND}"
+
+S=${WORKDIR}/${PN}
+DOCS=( README.md )
```

Comment:
(This is part of default einstalldocs array list with eapi >5)

Reply:
Great, time to remove the line then! :)